commitlint.config.ts

```diff
@@ -139,6 +143,32 @@ module.exports = {
                     return Plugins.properIssueRefs(rawStr);
                 },
 
+                "default-revert-message": (
+                    {
+                        header,
+                        body,
+                    }: {
+                        header: any;
+                        body: any;
+                    },
+                    when: string
+                ) => {
+                    let bodyStr = Helpers.convertAnyToString(body, "body");
+                    let headerUncastedStr = Helpers.convertAnyToString(
+                        header,
+                        "header"
+                    );
+                    let headerStr = Helpers.assertNotNull(
+                        headerUncastedStr,
```

Comment:
@realmarv let's inline this variable because it's not being used anywhere else

Reply:
What do you mean? Do you mean that I call Helpers.assertNotNull directly on the previous function call?

Reply:
when there is a variable "let foo=1+2; someFunc(foo);" and I ask you to inline foo, then you simply doo "someFunc(1+2)"

Reply:
Done